src/dpos-user.ts

```diff
@@ -394,20 +420,62 @@ export class DPOSUser {
     }
     signature = pendingReceipt.oracleSignature
 
+    log('Got receipt', pendingReceipt)
     return CryptoUtils.bytesToHexAddr(signature)
   }
 
   private async withdrawCoinFromRinkebyGatewayAsync(
     amount: BN,
     sig: string
   ): Promise<ethers.ContractTransaction> {
+    const hash = await this.createWithdrawalHash(amount)
```

Comment:
Hash is only used when `version == 2` so why is it computed when `version == 1`?

Reply:
Good point, thanks.